mmstudio/src/main/java/org/micromanager/acquisition/internal/acqengjcompat/AcqEngJAdapter.java

```diff
@@ -334,6 +378,24 @@ public static void addMMImageMetadata(JSONObject imageMD) {
             imageMD.put(PropertyKey.POSITION_INDEX.key(),
                   AcqEngMetadata.getAxisPosition(imageMD, "position"));
          }
+         if (imageMD.has("XPosition_um")) {
+            imageMD.put(PropertyKey.X_POSITION_UM.key(), AcqEngMetadata.getStageX(imageMD));
+         } else if (imageMD.has("XPosition_um_Intended")) {
+            imageMD.put(PropertyKey.X_POSITION_UM.key(), imageMD.getDouble(
+                  "XPosition_um_Intended"));
+         }
+         if (imageMD.has("YPosition_um")) {
+            imageMD.put(PropertyKey.Y_POSITION_UM.key(), AcqEngMetadata.getStageY(imageMD));
+         } else if (imageMD.has("YPosition_um_Intended")) {
+            imageMD.put(PropertyKey.Y_POSITION_UM.key(), imageMD.getDouble(
+                  "YPosition_um_Intended"));
+         }
+         if (imageMD.has("ZPosition_um")) {
+            imageMD.put(PropertyKey.Z_POSITION_UM.key(), AcqEngMetadata.getZPositionUm(imageMD));
+         } else if (imageMD.has("ZPosition_um_Intended")) {
+            imageMD.put(PropertyKey.Z_POSITION_UM.key(), imageMD.getDouble(
+                  "ZPosition_um_Intended"));
+         }
```

Comment:
It could be good to use variables from acqEngJ rather than writing out strings. i.e. `AcqEngMetadata.X_UM`, `AcqEngMetadata.X_UM_INTENDED`, etc

Or actually, you were adding "has" methods for everything in `AcqEngMetadata` so maybe that would be more consistent. Not a big deal either way though so long as it works

Reply:
I'll fix this in a second merge.  Still using an older AcqEngJ here.

Reply:
Realized first needed to back merge main branch, so fixed these now.